docs/system/system-metrics.md

```diff
@@ -11,6 +11,11 @@ metrics in OpenTelemetry. Consider the [general metric semantic
 conventions](/docs/general/metrics.md#general-metric-semantic-conventions) when creating
 instruments not explicitly defined in the specification.
 
+Note that `system.*` namespace should only be used for hosts' metrics.
+For resource attributes related to a host the `host.*` namespace should be used.
+The `system.*` namespace for metrics should be used only when the metrics are collected from within the target system.
```

Comment:
```suggestion
The `system.*` namespace should only be used when the metrics are collected from within the target system.
```
Can we maybe mention on example for "target system"? It might not be clear for everyone. 

Also: I would move this whole section below the initial paragrah "Note that `system.*`.. and leave the "resource attribute" part to the last.